test/prog.h

```diff
@@ -6,6 +6,15 @@
 #ifndef PROG_H
 #define PROG_H 1
 
+//Define Boolean type
```

Comment:
Defining `bool` and friends is not acceptable:

1. There is the C99 standard `<stdbool.h>`
2. The library does not use booleans.

If you need a boolean, use an `int`.

Reply:
I tried to keep it C98 compatible. If it is a problem I can just use int sure. Do you generally don't want typedef or just in this case?

Reply:
This is just to be similar to the rest of the code.  The library does not use explicit boolean types.